bigbluebutton-html5/private/config/settings.yml

```diff
@@ -351,6 +351,7 @@ public:
       # subscribe reattempt increases the reconnection timer up to this
       maxTimeout: 60000
     screenshare:
+      allowScreensharePresentationSwitch: false
```

Comment:
I don´t this the name should contain `ScreensharePresentation` since it is already under `screenshare`  configs.

what about `quickSwapEnabled` ?

Reply:
> Why not put it under the `layout` object as this is more of a layout feature rather than a screen sharing specific one?

That makes sense! But in that case, we should also move `screenshare.showButtonForNonPresenters` to `layout`.

My suggestion is to use the new flag `quickSwapEnabled` solely for layout decisions, without validating it on the backend. If we need an option to fully disable the feature, `disabledFeatures` would be a better fit.

If we move it to `layout` and keep it purely as a layout decision, a name like `layout.showScreenshareQuickSwapButton` would work well.